src/frontends/onnx/frontend/src/core/graph.cpp

```diff
@@ -384,10 +384,25 @@ OutputVector Graph::make_ng_nodes(const Node& onnx_node) {
         std::rethrow_exception(std::current_exception());
     }
 
+    size_t output_size = 0;
+    std::for_each(std::begin(ng_subgraph_outputs),
+                  std::end(ng_subgraph_outputs),
+                  [&output_size](const Output<ov::Node>& output) {
+                      output_size += output.get_node()->get_output_size();
```

Comment:
Unfortunately no (this was my first attempt), it doesn't work for case where impl return vector of nodes (like return {add}) instead of vector of outputs (like return add.outputs())